docs/get-started/how-to/gas-fees.mdx

```diff
@@ -67,19 +67,38 @@ On Linea, it's used by the sequencer and Linea Besu nodes running the correct pl
 
 Variable cost is calculated with the following formula:
 
-```python
-VARIABLE_COST (4 bytes) = min(max(
-(
-  (
-    ((averageWeightedBaseFee + averageWeightedPriorityFee) * 
-    blob-submission-expected-execution-gas + averageWeightedBlobBaseFee * expected-blob-gas
-  ) / bytes-per-data-submission) * profit-margin
-)
-, min-bound), max-bound)
+```bash
+ADJUSTED_VARIABLE_COST = max(base_variable_cost, previous_variable_cost * ( 1 + delta / CHANGE_DENOMINATOR )
 ```
 
-The `profit-margin` ensures the network is sustainable. `min-bound` and `max-bound` are variable, 
-and guarantee the gas price stays within a reasonable range.
+Where:
+- `base_variable_cost` is calculated with the below formula:
+  ```python
+  variable_cost (4 bytes) = min(max(
+  (
+    (
+      ((averageWeightedBaseFee + averageWeightedPriorityFee) * 
+      blob-submission-expected-execution-gas + averageWeightedBlobBaseFee * expected-blob-gas
+    ) / bytes-per-data-submission) * profit-margin
+  )
+  , min-bound), max-bound)
+  ```
+  The `profit-margin` ensures the network is sustainable. `min-bound` and `max-bound` are variable, 
+  and guarantee the gas price stays within a reasonable range.
+- `previous_variable_cost` is the value of the previous block's `ADJUSTED_VARIABLE_COST` calculation
```

Comment:
More percisely, we can just say "... is the value of the last `ADJUSTED_VARIABLE_COST` calculation" instead of `previous block's`